### ISSUE TITLE
Changing to MSE instead of MeanError

### ISSUE THREAD

**Issue by JonasFrey96** [leggedrobotics/less-is-more#2]
small bug in here for validation metrics:
https://github.com/leggedrobotics/less-is-more/blob/eee7592448b65e60695f7b31d6acf914e8991e96/limo/src/models/limo_model.py#L30-L32

**Comment by manthan99** [comment]
This is correct as we are just logging the metric and the loss is defined in the config as MSE.

**Comment by yv1es** [comment]
I agree with Manthan on this. The `MeanMetric` is used to aggregate the training loss over the batches in an epoch. 
MSE is defined as our training loss in  [limo/configs/model/limo.yaml](https://github.com/leggedrobotics/less-is-more/blob/eee7592448b65e60695f7b31d6acf914e8991e96/limo/configs/model/limo.yaml).
